### PR TITLE
Revert "FIX_SCRUB option for users"

### PR DESCRIPTION
**Pull request by jkuang-exploro**
This reverts commit 671bca00382ea09f0b853051994c9926cfd17114.

### CODE REVIEW

**Review by guillaumevernieres — APPROVED** [review]
All good, just tested 1 cycle (). Let's wait for @flampouris before you merge this @JianKuang-NOAA .

File: workflow/user.yaml.default
Comment (flampouris) [inline workflow/user.yaml.default:3]: If a path is not set here, what's happening?

@JianKuang-NOAA please add the description of the SCRUB to the README.

I will include that in the PR_ens request.

**Review by flampouris — CHANGES_REQUESTED** [review]

**Review by jkuang-exploro — COMMENTED** [review]
I will add description in PR_ens, another pull request that re-introduce the FIX_SCRUB feature. 

This is the bug fix request.

**Comment by flampouris** [comment]
Ok, then I am going forward with the merging.

**Comment by guillaumevernieres** [comment]
> I will add description in PR_ens, another pull request that re-introduce the FIX_SCRUB feature.
> 
> This is the bug fix request.

:+1:  @flampouris  , we're only going back to a working commit

**Comment by flampouris** [comment]
Sorry everyone, my mind was still on the pr_ens

**Pull request merged**
